src/data/battle-anims.ts

```diff
@@ -267,7 +267,7 @@ export abstract class BattleAnim {
                   spriteSource!.frame.name,
                   true,
                 ); // TODO: are those bangs correct?
-                ["spriteColors", "fusionSpriteColors"].map(
+                ["spriteColors"].map(
                   (k) => (sprite.pipelineData[k] = (isUser ? user! : target).getSprite().pipelineData[k]),
                 ); // TODO: are those bangs correct?
```

Comment:
```suggestion
                sprite.pipelineData["spriteColors"] = (isUser ? user! : target).getSprite().pipelineData["spriteColors"]; // TODO: are those bangs correct?
```
(...and similarly for the other `["spriteColors"].map` blocks of code)